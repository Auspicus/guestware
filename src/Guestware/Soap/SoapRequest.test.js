import 'regenerator-runtime/runtime'

import { SoapRequest } from '../../../lib/index.js'

describe('SoapRequest', () => {

  test('.toString => returns XML body', () => {
    expect(new SoapRequest('<xml></xml>').toString()).toMatchSnapshot()
  })

})