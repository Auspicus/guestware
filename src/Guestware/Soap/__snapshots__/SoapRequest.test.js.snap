// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SoapRequest .toString => returns XML body 1`] = `"<xml></xml>"`;
